Makefile

```diff
@@ -186,12 +186,31 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) ./hack/multikueue-e2e-test.sh
 
+.PHONY: install-shellcheck
+install-shellcheck: 
+	# removing previous shellcheck files if shellcheck give errors and warnings it wouldn't remove the downloaded files
+	rm -rf shellcheck-v0.9.0*
+	@echo "Installing ShellCheck"
+	curl -s https://api.github.com/repos/koalaman/shellcheck/releases/latest \
+    | grep "browser_download_url.*linux.x86_64.tar.xz" \
```

Comment:
Can you obtain the OS and CPU arch from go env, and then download the binary for dedicated the OS and CPU arch?

Reply:
we can use https://github.com/dvershinin/lastversion to download the latest stable release. It downloads the binary release specific to the OS.

Reply:
I don't think that we should use the latest version. We should use a specified version as well as other tools.
Here, the specified version means v0.9.0.

Reply:
It obtains the most recent stable release. When I tested it, it downloaded v0.9.0 automatically. 
Downloading the OS-specific stable release.

Reply:
The utility downloads the latest stable version not the stable version. I tried getting the URL according to the OS and architecture but it didn't work.